src/scss/partials/_form.scss

```diff
@@ -0,0 +1,33 @@
+.login_pic {
+  width: 100%;
+  max-width: 500px;
+  padding: 25px;
+}
+
+p {
+  width: 100%;
+  max-width: 750px;
+  margin: 0 auto;
+}
+
+.mb-4 {
+  padding: 10px;
```

Comment:
This is a big problem. Margin classes are for margins only. Not padding, that's what padding classes are for.

Reply:
i have done same changes in log in page